.github/workflows/ci.yaml

```diff
@@ -51,6 +51,25 @@ jobs:
       - run: |
           cd mirrord-protocol
           cargo test
+
+  test_mirrord_config:
```

Comment:
Is this supposed to be part of this PR?

Reply:
nope this is from dima's pr, probably came along because of the merge resolution?